internal/providers/terraform/azure/vpn_gateway_connection.go

```diff
@@ -0,0 +1,75 @@
+package azure
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+	"github.com/tidwall/gjson"
+)
+
+func GetAzureRMVpnGatewayConnectionRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_virtual_network_gateway_connection",
+		RFunc: NewAzureRMVpnGatewayConnection,
+		ReferenceAttributes: []string{
+			"type",
+		},
+		Notes: []string{"Price for additional S2S tunnels is used"},
+	}
+}
+
+func NewAzureRMVpnGatewayConnection(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+
+	sku := "Basic"
+
+	var vpnGateway *schema.ResourceData
+	if len(d.References("type")) > 0 {
+		vpnGateway = d.References("type")[0]
+		sku = vpnGateway.Get("sku").String()
+	}
+
+	region := lookupRegion(d, []string{})
+
+	meterName := sku
+
+	costComponents := make([]*schema.CostComponent, 0)
+
+	if sku == "Basic" {
+		sku = "Basic Gateway"
+		meterName = "Basic"
+	}
```

Comment:
```suggestion

```
It looks like the S2S price record has `"skuName": "Basic"`

Reply:
tf has sku Basic but in pricing data its Basic Gateway